cmd/stack/endpoints.go

```diff
@@ -37,7 +37,7 @@ func Endpoints(ctx context.Context) *cobra.Command {
 		Use:   "endpoints [service...]",
 		Short: "Show endpoints for a stack",
 		RunE: func(cmd *cobra.Command, args []string) error {
-
+			oktetoLog.Warning("'okteto stack endpoints' is deprecated and will be removed in a future version")
```

Comment:
should we have an `okteto endpoints` command? let's create an issue for it, but I don't think we need it for now